crates/kona-providers/src/lib.rs

```diff
@@ -9,3 +9,6 @@ extern crate alloc;
 
 /// Re-export kona's derivation traits
 pub use kona_derive::traits::*;
+
+pub mod chain_provider;
+pub use chain_provider::{InMemoryChainProvider, InMemoryChainProviderInner};
```

Comment:
I think the Inner could also remain private since it doesn't impl the ChainProvider